Tizen Studio Hot Reload Feature (#1155)
Samsung/tizen-docs: docs/application/dotnet/get-started/wearable/first-xaml-app.md

```bash
$ sdb install <path-to-package>/org.tizen.example.TizenWearableXamlApp1-1.0.0.tpk
```

> ## Validate Your Application Changes Using XAML Hot Reload

Reviewer: Sorry for the missing the title checking.

As we know, we are going to use sentence-style capitalization.

However to get the consistency, in this page it is better to use capital letter for the every 1st letters, as like other titles.
(I think the titles will be updated in a lump in the future.)

But... Does the "Using" also require capital letter? Would you please let me know your opinion?

Author: Will using lower case for the word using.

> To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, you can now immediately see the UI edits on the running emulator without having to rebuild or redeploy your app. This feature helps you save time and validate your changes easily.

Reviewer 2: app->application

Author: @sijupunnoose,

Thank you for your review. I will do the needful.

> To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, you can now immediately see the UI edits on the running emulator without having to rebuild or redeploy your application. This feature helps you save time and validate your changes easily.

Reviewer: ```suggestion
To test your UI code changes in your application, you must build and deploy your application. With the introduction of the XAML Hot Reload feature, the UI edits on the running emulator are applied immediately on the running emulator without rebuild or deploying your application again. It helps you save time and validate the changes easily.
```

However, this suggestion is related with https://github.com/Samsung/tizen-docs/pull/1155#discussion_r448718974, and I'm waiting the answer to the question to this comment.

Author: @mijin85cho,
As mentioned in the previous comment, if I change "would have" to to must, then in that case we are first telling the user that we "must do" a certain activity. Then, in the immediate next sentence we are saying that now because of the new feature you need not do that activity. So here the usage of must according to me is contradicting and may not give the right message.
If you still suggest to use must, I will change it to "must".

Please let me know if the following is okay:

To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, the UI edits are immediately applied on the running emulator without rebuild or deploying your application again. It helps you save time and validate the changes easily.

Please let me know your views.

Reviewer: @surbhig12 
Thanks for the explanation. I understood your worrying point.
Considering your concern, how about below style?

To test your UI code changes in your application, you must build and deploy your application. However, if you use XAML Hot Reload feature, the UI edits are immediately applied on the running emulator without rebuild or deploying your application again. It helps you save time and validate the changes easily.

Author: @mijin85cho,

Thank you for understanding.
I suggest:

To test your UI code changes in your application, you must build and deploy your application. However, if you use the XAML Hot Reload feature, the UI edits are immediately applied on the running emulator without building or deploying your application again. It helps you save time and validate the changes easily.

Please verify

Reviewer: I agree with your rewriting. Thanks for the modification.

> To use the Hot Reload feature, you must enable it in your Visual Studio IDE. To enable:

Reviewer: ```suggestion
To use the Hot Reload feature, enable it in your Visual Studio IDE. To enable the Hot Reload feature, follow these steps:
```

Author: @mijin85cho,

I did not suggest this change as I wanted to make the sentence crisp and easy to read. However, I do understand that we need to maintain consistency. I will accept your comment.

To use the Hot Reload feature, enable it in your Visual Studio IDE. To enable the Hot Reload feature, follow these steps:

1. In the Visual Studio menu, select **Tools** > **Options**.
2. In the **Options** window that appears, select **Tizen** > **Tools**.
3. Select **Enable XAML Hot Reload**.

    ![Enable Hot Relaod](media/vs_enable_hot_reload.PNG)

### Using XAML Hot Reload 

You can launch the instance in debugging mode and run the application by clicking the emulator instance in the Visual Studio toolbar.

![Deploy your package](media/vs_emulator_launch_xaml.png)

After you make UI changes in your application code, you do not need to rebuild or redeploy the application.

> You will see that your changes are immediately reflected on your emulator while you are in debugging mode.

Reviewer: @surbhig12 
One question about the tone of the sentences.

In the updated parts, I can see the tone like "you will see", "you can now immediately see", "you would have to".
It gives me more familiar impression than other parts,
as at the other parts, "~ appears/changes", "you must" tone is used.

So, to me, this parts gives me the impression of different tone compared to left.
Isn't it better to use unified tone in one page? Would you please give me your opinion to my question?

Author: @mijin85cho,
In the newly added part the tone has been kept similar to the tone in the current page. However, you may feel that in certain sentences the tone is different. For example, 
for the sentence: To test your UI code changes in your application, you would have to build and deploy your application.

If we change the sentence to "To test your UI code changes in your application, you must build and deploy your application. With the introduction of the XAML...", then the flow and meaning changes.

Therefore, to avoid changing the meaning, we can keep it as is.
Let me know your views.

Reviewer: > If we change the sentence to "To test your UI code changes in your application, you must build and deploy your application. With the introduction of the XAML...", then the flow and meaning changes.

Would you please let me the changes in detail? What flow and meaning are changed?

> You will see that your changes are immediately reflected on your emulator while you are in debugging mode.

Reviewer: ```suggestion
While you are in debugging mode, your changes are immediately reflected on your emulator.
```

Author: I agree

> [!Note]

> > The XAML Hot Reload feature is available for Tizen XAML App and Tizen Wearable XAML App.

Author: Thank you for pointing out. I will fix it.
